thorlcr/graph/thgraph.cpp

```diff
@@ -1727,6 +1728,10 @@ void CGraphBase::abort(IException *e)
 
         abortException.set(e);
         aborted = true;
+
+        // prevent dtors throwing ... __verbose_terminate_handler()->abort()->raise()->core files
+        setJobAborted(1);
```

Comment:
I wouldn't set in clean(), because we still want to see truly unexpected crashes and their cores, e.g. if job is finishing in an otherwise successful state and crashes as it's winding up.

If on the other hand, it's winding up after a reported aborting condition and throws an unhandled exception that leads to a core, we care much less.

So I would turn this on, only when know there's a abort type scenario in motion.
There may not be 1 place that covers all bases though, some places where it would be good to disable potential follow-on unhanded exceptions are:

+ When the slave receives a GraphAbort: message (slavmain.cpp).
+ When the slave has reported an exception to the master. There are 2 places that happens a) CJobSlaveChannel::fireException, and b) in slavmain.cpp CThreadExceptionCatcher::fireException
+ When the Thor CJobManger::stop() has been called.

Reply:
I misread the PR, thought above change was in clean(), I see it's in abort() now.
Which is good catch all place for any abort handling, so I would leave as is.
It might be a little late in some cases, but I think good enough.

Reply:
After offline discussion leaving this in abort() is ok.